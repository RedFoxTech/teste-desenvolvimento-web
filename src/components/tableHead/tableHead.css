.arrow {
    width: 15px;
    margin: 2px;
    cursor: pointer;
}