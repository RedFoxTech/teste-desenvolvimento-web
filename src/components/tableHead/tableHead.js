import React from 'react';

import './tableHead.css';

import arrowUp from '../../assets/arrow_up.png';
import arrowDown from '../../assets/arrow_down.png';

const TableHead = (props) => {
    return (
        <div className="d-flex justify-content-around align-items-center">
            {(props.shownText || props.id)}
            <div>
                <img src={arrowUp} className="arrow ml-3" alt="arrow up" onClick={(evt) => props.sortList(props.id, "up")}/>
                <img src={arrowDown} className="arrow" alt="arrow down" onClick={(evt) => props.sortList(props.id, "down")}/>
            </div>
        </div>
    )
}

export default TableHead;