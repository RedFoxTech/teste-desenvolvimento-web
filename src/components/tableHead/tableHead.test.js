import React from 'react';
import tableHead from './tableHead';

let wrapper;
beforeEach(() => {
    wrapper = shallow(<tableHead/>);
})

describe("rendering test/snapshot", () => {
    it('should render the proper header prototype', () => {
        expect(wrapper).toMatchSnapshot()
    })
})