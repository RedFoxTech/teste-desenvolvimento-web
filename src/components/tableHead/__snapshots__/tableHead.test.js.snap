// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering test/snapshot should render the proper header prototype 1`] = `<tableHead />`;
